Create a test for Response class

siplowski opened the pull request.

siplowski committed:
Create a test for Response class

Edit Carbon assertions.
Edit uses.

Edit carbon instantiations

Complete coverage for a Provider class

Add exception code and text expectation.
Remove excess response assertions.

Add phpdoc

sashabeton commented on tests/Authorization/ResponseTest.php:
```diff
@@ -0,0 +1,60 @@
+
+class ResponseTest extends TestCase
+{
+
```
Не хватает phpdoc с типом

sashabeton commented on tests/Authorization/ResponseTest.php:
```diff
@@ -0,0 +1,60 @@
+    public function setUp()
+    {
+        parent::setUp();
+
```
Нет смысла объявлять setUp(), если в классе один тест

sashabeton replied:
Соответственно, не нужно будет объявлять лишнее свойство

Horat1us commented on tests/Authorization/ResponseTest.php:
```diff
@@ -0,0 +1,70 @@
+    protected $response;
+
+    /**
+     * @return void
```
Зачем тут phpDoc?

https://github.com/wearesho-team/php-code-conventions#-В-phpdoc-в-возвращаемом-значении-не-надо-указывать-void-и-null

Его нужно указать в типе функции 
```php
<?php
function setUp(): void 
{}
```

Horat1us commented on tests/Authorization/ResponseTest.php:
```diff
@@ -0,0 +1,70 @@
+    }
+
+    /**
+     * @return void
```
Этот phpDoc тоже не нужен

sashabeton commented on tests/Authorization/ResponseTest.php:
```diff
@@ -0,0 +1,64 @@
+    /** @var Ubki\Authorization\Response */
+    protected $response;
+
+    public function setUp():void
```
setUp(): void
Необходимо ставить пробел между двоеточием и типом

sashabeton commented on tests/Authorization/ResponseTest.php:
```diff
@@ -0,0 +1,64 @@
+        );
+    }
+
+    public function testGets():void
```
Аналогично

siplowski committed:
Refactor test

Refactor test

Remove return type phpdocs. Add return types.

Spaces added before return types

remove phpdoc

remove phpdoc

remove phpdoc

remove phpdoc

Horat1us commented on tests/Authorization/ResponseTest.php:
```diff
@@ -0,0 +1,58 @@
+
+    public function testGets(): void
+    {
+        /** @var Ubki\Authorization\Response */
```
Зачем этот комментарий?

Horat1us reviewed (changes requested):
Необходимо сделать rebase коммитов

Horat1us committed:
Merge branch 'master' into tests/response
